dashboard/app/views/layouts/_school_info_confirmation_dialog.html.haml

```diff
@@ -16,14 +16,14 @@
 - script_data[:authTokenValue] = form_authenticity_token
 - script_data[:existingSchoolInfo] = {}
 - if current_user_school_info
-  - script_data[:existingSchoolInfo] = current_user_school_info.slice(:id, :country, :school_type, :school_id, :school_name, :full_address)
+  - script_data[:existingSchoolInfo] = current_user_school_info.slice(:country, :school_id, :school_name, :school_type)
 - if current_user_user_school_info
   - script_data[:existingSchoolInfo][:user_school_info_id] = current_user_user_school_info.id
-- script_data[:existingSchoolInfo][:country] ||= 'United States' if us_ip
+- script_data[:existingSchoolInfo][:country] ||= 'US' if us_ip
 - script_data[:existingSchoolInfo][:school_name] ||= current_user_school_info&.school&.name
-- script_data[:mapboxAccessToken] = CDO.mapbox_access_token
+- script_data[:existingSchoolInfo][:school_zip] ||= current_user_school_info&.school&.zip || current_user_school_info&.zip&.to_s&.rjust(5, '0')
```

Comment:
Kind of hard to tell what's happening here.. So when an nces school with all its info is associated with a teacher, we use that zip, which is stored as a string. When a teacher can't find their school, or they don't teach in a school setting, we store the zip they provide, but the school_info.zip column is an integer instead of a string. Rather than running a migration to fix this column, I'm converting to a string and padding the start with zeros 🤷